core/src/com/unciv/models/ruleset/unique/Unique.kt

```diff
@@ -225,7 +225,7 @@ class UniqueMapTyped: EnumMap<UniqueType, ArrayList<Unique>>(UniqueType::class.j
 
 
 // Will probably be allowed to be used as a conditional when I get the motivation to work on that -xlenstra
-class TemporaryUnique() {
+class CivwideUnique() {
```

Comment:
What is the reason this is renamed?

Reply:
A part of my nefarious plan to use this for permanent conditionals as well.
I'm still planning that but this change should go in a different pr :)